spec/lib/query/proposal/search_spec.rb

```diff
@@ -83,7 +83,7 @@
       user = create(:user, client_slug: "test")
 
       proposal1 = create(:proposal, id: 199, requester: user)
-      test_client_request1 = create(:test_client_request, proposal: proposal1)
+      create(:test_client_request, proposal: proposal1)
```

Comment:
I don't have that plugin so it hasn't bothered me.

I guess I wrote enough C that the idea of ignoring the return value (because memory leak) just got ingrained. Since they aren't used, I don't care if you leave it as you've got it.